Testscripts/Linux/SRIOV-VerifyVF-Connection.sh

```diff
@@ -33,6 +33,15 @@ if [ $? -ne 0 ]; then
     exit 0
 fi
 
+if [ "$rescind_pci" = "yes" ]; then
+    # call rescind function with param SRIOV
+    if ! RescindPCI "SR-IOV"; then
```

Comment:
can you use check_exit_status here instead of the whole if lines?

Reply:
`check_exist_status` sets the test to aborted if instructed to exit, where as for this case we want it to fail as the for the rescind case, this is the beginning of the test. And if we don't exit on `check_exit status` we're just adding extra lines here.